e2e/testdata/porch/rpkg-lifecycle/config.yaml

```diff
@@ -158,6 +158,9 @@ commands:
       - --namespace=rpkg-lifecycle
     stderr: |
       cannot reject git-017a8366a5e0d9b35ae6dc489d4d3f68046d6034 with lifecycle 'Published'
+      Error: errors:
```

Comment:
This output is a bit awkward, but it isn't really due to this PR. We should fix that in a separate PR.